README.md

```diff
@@ -117,6 +117,31 @@ If you use `-skip-dnssec`, letsdane will use the Authenticated Data flag.
 
 I wanted to try DANE, but no browser currently supports it. It may still be a long way to go for browser support, but if you want to try it now you can!
 
+
+## Docker Usage
+
+### Docker CLI

```

Comment:
You can put Docker usage after Handshake section so it's easier to notice. For now you can instruct users to build their own image using `docker build -t letsdane .` and use that in the docker run example (i'm planning to maintain images with tagged versions on docker hub but not at the moment).